zipline/utils/events.py

```diff
@@ -603,6 +605,11 @@ class time_rules(object):
     every_minute = Always
 
 
+class calendars(object):
+    US_EQUITIES = sentinel('us_equities_calendar')
```

Comment:
Most of our uses of `sentinel` right now use the same string for both the variable name and the repr.  Any reason not to do that here?